### ISSUE TITLE
How does a forced reply bot get messages sent by users

### ISSUE THREAD

**Issue by flllkis** [eternnoir/pyTelegramBotAPI#1781]
How does a forced reply bot get messages sent by users

**Comment by coder2020official** [comment]
Please use discussions next time. 
pass [ForceReply](https://pytba.readthedocs.io/en/latest/types.html#telebot.types.ForceReply) to reply_markup in your method.

**Comment by flllkis** [comment]
Can you give me a simple example

**Comment by flllkis** [comment]
> 请下次使用讨论。[在您的方法中将 ForceReply](https://pytba.readthedocs.io/en/latest/types.html#telebot.types.ForceReply) 传递给 reply_markup。[](https://pytba.readthedocs.io/en/latest/types.html#telebot.types.ForceReply)

Can you give me a simple example

**Comment by coder2020official** [comment]
bot.send_message(id, text, reply_markup=telebot.types.ForceReply()) afaik smth like this

**Comment by flllkis** [comment]
> bot.send_message(id, text, reply_markup=telebot.types.ForceReply()) afaik smth like this

No, I mean how do I get the mandatory message content of the client reply

**Comment by flllkis** [comment]
> bot.send_message(id, text, reply_markup=telebot.types.ForceReply()) afaik smth like this

If my mandatory message is your name and the client replies to zhengling, how do I get the zhengling content
